core-build/gulp-core-build-serve/README.md

```diff
@@ -121,7 +121,7 @@ The certificate is generated and self-signed with a unique private key and an at
 it (Windows and macOS only). If the user does not agree to trust the certificate, provides invalid root
 credentials, or something else goes wrong, the `TrustCertTask` will fail and the `ServeTask` will serve
 with the default, non-trusted certificate. If trust succeeds, the certificate and the private key are
-dropped in the `.gcb-serve-data` directory in the user's home folder in `PEM` format. On platforms
+dropped in the `.rushstack-serve-data` directory in the user's home folder in `PEM` format. On platforms
```

Comment:
Let's rename this from `rushstack-serve-data` to something like just `.rushstack`.